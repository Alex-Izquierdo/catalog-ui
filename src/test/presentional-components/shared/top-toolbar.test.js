import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import TopToolbar from '../../../PresentationalComponents/Shared/top-toolbar';

describe('<TopToolbar />', () => {
  it('should render correctly', () => {
    expect(toJson(mount(<TopToolbar />))).toMatchSnapshot();
  });
});
