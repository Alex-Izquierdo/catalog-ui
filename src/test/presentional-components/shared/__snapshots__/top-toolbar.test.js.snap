// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopToolbar /> should render correctly 1`] = `
<TopToolbar
  paddingBottom={true}
>
  <div
    className="pf-u-pt-xl pf-u-pr-xl pf-u-pl-xl pf-u-pb-xl top-toolbar"
  />
</TopToolbar>
`;
